app/experimenter/experiments/migrations/0163_auto_20210325_2041.py

```diff
@@ -0,0 +1,79 @@
+# Generated by Django 3.1.7 on 2021-03-25 20:41
+
+from django.db import migrations, models
+
+from experimenter.experiments.constants import NimbusConstants
+
+
+def update_changelogs(apps, schema_editor):
+    NimbusChangeLog = apps.get_model("experiments", "NimbusChangeLog")
+    db_alias = schema_editor.connection.alias
+    NimbusChangeLog.objects.using(db_alias).filter(old_status="Review").update(
+        old_status=NimbusConstants.Status.DRAFT.value,
+        old_publish_status=NimbusConstants.PublishStatus.REVIEW.value,
+    )
+    NimbusChangeLog.objects.using(db_alias).filter(old_status="Accepted").update(
+        old_status=NimbusConstants.Status.DRAFT.value,
+        old_publish_status=NimbusConstants.PublishStatus.WAITING.value,
+    )
+    NimbusChangeLog.objects.using(db_alias).filter(new_status="Review").update(
+        new_status=NimbusConstants.Status.DRAFT.value,
+        new_publish_status=NimbusConstants.PublishStatus.WAITING.value,
+    )
+    NimbusChangeLog.objects.using(db_alias).filter(new_status="Accepted").update(
+        new_status=NimbusConstants.Status.DRAFT.value,
+        new_publish_status=NimbusConstants.PublishStatus.WAITING.value,
+    )
```

Comment:
@jaredlockhart your comment disappeared since I fixed up the other commits (kept running into conflicts with Jody) but thank you for bearing with me 🤞